### DISCUSSION TITLE
nvidia GPU as a detector?

### DISCUSSION THREAD

**Discussion by bbrendon** [blakeblackshear/frigate#783] [Ask A Question] [answered]
It seems you can't use nvidia GPU as a detector? only CPU or coral? Is there a plan to add nvidia or is nvidia not really helpful for that?

**Comment by blakeblackshear** [comment, marked as answer]
https://github.com/blakeblackshear/frigate/issues/659

**Reply by bbrendon** [reply]
How did I miss that? I think I have a hard time spelling nvidia sometimes :) Thanks.
